roles/ceilometer-common/tasks/main.yml

```diff
@@ -14,7 +14,7 @@
         owner=ceilometer group=ceilometer
 
 - name: ceilometer config
-  template: src=etc/ceilometer/{{ item }} dest=/etc/ceilometer/{{ item }} mode=0644
+  template: src=etc/ceilometer/{{ item }} dest=/etc/ceilometer/{{ item }} mode={{ 0644 if 'policy.json' in item else 0640 }} owner=ceilometer group=ceilometer
```

Comment:
if policy needs a different mode I'd rather it have it's own task.    why does it need to be different to the others ?